shared/patches/recyclerlistview+4.2.0.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js b/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
+index 9cd3c57..f65012e 100644
+--- a/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
++++ b/node_modules/recyclerlistview/dist/reactnative/core/layoutmanager/LayoutManager.js
+@@ -114,7 +114,7 @@ var WrapGridLayoutManager = /** @class */ (function (_super) {
+         for (var i = startIndex; i < itemCount; i++) {
+             oldLayout = this._layouts[i];
+             var layoutType = this._layoutProvider.getLayoutTypeForIndex(i);
+-            if (oldLayout && oldLayout.isOverridden && oldLayout.type === layoutType) {
++            if (oldLayout && oldLayout.isOverridden /*&& oldLayout.type === layoutType*/) {
+                 itemDim.height = oldLayout.height;
+                 itemDim.width = oldLayout.width;
+             }
+@@ -145,9 +145,12 @@ var WrapGridLayoutManager = /** @class */ (function (_super) {
+                 itemRect.x = startX;
+                 itemRect.y = startY;
+                 itemRect.type = layoutType;
++		const nextOverriden = !!itemRect.isOverridden && itemRect.width === itemDim.width && itemRect.height === itemDim.height;
```

Comment:
If we happened to adopt the size and have the same dimensions we can keep the override flag or we don't have it